service/src/test/java/org/jboss/sbomer/service/test/integ/feature/sbom/SBOMResourceRSQIT.java

```diff
@@ -162,10 +166,7 @@ void testFetchSbomByPurlNotFound() {
                         .statusCode(404)
                         .body("errorId", CoreMatchers.isA(String.class))
                         .body("error", CoreMatchers.is("Not Found"))
-                        .body(
-                                "message",
-                                CoreMatchers.is(
-                                        "SBOM with purl = 'pkg:maven/org.apache.logging.log4j/log4j@2.19.0.redhat-00001?type=pom' couldn't be found"))
+                        .body("message", CoreMatchers.is("SBOM with purl = 'pkg%3Amaven%2Forg.apache.logging.log4j%2Flog4j%402.19.0.redhat-00001%3Ftype%3Dpom' couldn't be found"))
```

Comment:
This expectation is wrong. What SBOMer returns (and should return) is exactly what it was before the proposed change. I wouldn't change this.